pkg/sql/plpgsql/parser/BUILD.bazel

```diff
@@ -35,6 +35,7 @@ go_library(
     importpath = "github.com/cockroachdb/cockroach/pkg/sql/plpgsql/parser",
     visibility = ["//visibility:public"],
     deps = [
+        "//pkg/build",
```

Comment:
idk why this change was made by `dev gen`...

Reply:
From the failed run it looks like it needs to run dev gen bazel again. Maybe that will help?